### PR TITLE
Chore: Publishing setup

### PR DESCRIPTION
**Pull request by thomasdigby**
- Set initial version to 0.1.0 alpha
- Fixed test and dev dependency
- Specified output files for package
- Removed source maps

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -15,10 +15,10 @@ const peerDeps = Object.keys(pkg.peerDependencies || {})
 
 export default {
   input: 'src/index.ts',
-  external: [...deps, ...peerDeps],
+  external: [...deps, ...peerDeps, 'react-player/vimeo'],
   output: [
-    { file: pkg.main, format: 'cjs', sourcemap: true },
-    { file: pkg.module, format: 'esm', sourcemap: true }
+    { file: pkg.main, format: 'cjs' },
+    { file: pkg.module, format: 'esm' }
```
Comment (andoulla) [inline rollup.config.js:21]: how come we don't need the sourcemaps anymore? what am I missing here?

Comment (thomasdigby) [reply rollup.config.js:21]: It's unnecessary for third-party libs, we wouldn't (shouldn't) see the source code for an external package within the exported source maps for first party code.

Comment (andoulla) [reply rollup.config.js:21]: oh yes, why didn't I think of it? thanks

File: tsconfig.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "compilerOptions": {
     "outDir": "./dist/", // path to output directory
-    "sourceMap": true, // allow sourcemap support
+    "sourceMap": false, // allow sourcemap support
```
Comment (hollg) [inline tsconfig.json:4]: This defaults to `false` so we can remove the whole line. Alternatively, if we want to be explicit about not including source maps then I think we should update the comment on this line

**Review by hollg — APPROVED** [review]

**Pull request merged**